bin/src/command/orders.rs

```diff
@@ -806,45 +775,41 @@ impl CommandServer {
 
         return_processing(
             self.command_tx.clone(),
-            request_identifier.clone(),
+            client_id.clone(),
             "Reloading configuration, sending config messages to workers...",
         )
         .await;
 
-        for message in new_config.generate_config_messages() {
-            if let CommandRequestOrder::Proxy(order) = message.order {
-                if self.state.dispatch(&order).is_ok() {
-                    diff_counter += 1;
+        for order in new_config.generate_config_messages() {
+            if self.state.dispatch(&order.content).is_ok() {
+                diff_counter += 1;
 
-                    let mut found = false;
-                    let id = format!(
-                        "LOAD-STATE-{}-{}",
-                        &request_identifier.request, diff_counter
-                    );
+                let mut found = false;
+                let id = format!("LOAD-STATE-{}-{}", &order.id, diff_counter);
 
-                    for ref mut worker in self.workers.iter_mut().filter(|worker| {
-                        worker.run_state != RunState::Stopping
-                            && worker.run_state != RunState::Stopped
-                    }) {
-                        let worker_message_id = format!("{}-{}", id, worker.id);
-                        worker.send(worker_message_id.clone(), *order.clone()).await;
-                        self.in_flight
-                            .insert(worker_message_id, (load_state_tx.clone(), 1));
+                for ref mut worker in self.workers.iter_mut().filter(|worker| {
+                    worker.run_state != RunState::Stopping && worker.run_state != RunState::Stopped
```

Comment:
Could be refactor as:
```rust
.filter(|worker| !matches!(worker.run.state, RustState:: Stopping | RunState::Stopped))
```

Reply:
Yes, but in that case, could we name the method `is_running` ?

Reply:
yes!